pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.isort]
+profile="black"
+line_length = 120
+skip = ["venv"]
+
+[tool.black]
+line-length = 120
```

Comment:
Actually, the contributing guide for CNaaS-NMS specifies a line-length of 99 as policy. This should comply with that (unless SUNET would like to change their policy).